app/utils/callback/lava.py

```diff
@@ -222,7 +226,37 @@ def _get_lava_meta(meta, job_data):
             handler(meta, step["metadata"])
 
 
-def _add_test_log(meta, job_log, base_path, suite):
+def _get_dir_path(meta, base_path):
+    """Parse the meta-data from LAVA
```

Comment:
Looks like this docstring summary was left from a copy-paste and does not reflect what the function `_get_dir_path` actually does.